src/patternfly/components/DataToolbar/docs/code.md

```diff
@@ -0,0 +1,56 @@
+## Overview
+
+As the data toolbar component is a hybrid layout and component, some of its elements are presentational, while some require accessibility support.
+
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-data-toolbar` | `<div>` | Initiates the toolbar component. **Required** |
+| `.pf-c-data-toolbar__item` | `<div>` | Initiates the toolbar component item. **Required** |
+| `.pf-c-data-toolbar__group` | `<div>` | Initiates the toolbar component group. |
+| `.pf-c-data-toolbar__content` | `<div>` | Initiates the toolbar component content. |
+| `.pf-c-data-toolbar__expandable-content` | `<div>` | Initiates the toolbar component expandable content. |
+| `.pf-m-expanded` | `.pf-c-data-toolbar__expandable-content` | Modifies the component for the expanded state. |
+| `.pf-c-data-toolbar__divider` | `<div>` | Initiates the toolbar component divider. |
+| `.pf-m-bulk-select` | `.pf-c-data-toolbar__item` | Modifies toolbar item spacing. |
```

Comment:
Usually we'll put a description of what it modifies, like "modifies bulk select toolbar item spacing"